partitura/score.py

```diff
@@ -346,12 +344,10 @@ def metrical_position_map(self):
                 axis=0,
                 kind="linear",
                 fill_value="extrapolate",
+                dtype=int,
             )
 
-            def zero_fun(input):
```

Comment:
these lines were removed since this case is covered by the new `interp1d` utility in `partitura.utils.generic`.